ui/perfherder/CompareSelectorView.jsx

```diff
@@ -0,0 +1,245 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { react2angular } from 'react2angular';
```

Comment:
The default export of `react2angular` has been compiled down to ES5 - so for jobs-view we used the separate ES6 export (`react2angular/index.es2015`) instead (which is smaller).